### ISSUE TITLE
PersonaBar Themes blank when Importing Site Template with default theme that is not installed

### ISSUE THREAD

**Issue by morganalori** [dnnsoftware/Dnn.Platform#4147]
## Description of bug
Persona Bar Themes blank when importing a Site Template that has a default theme set that is not installed on the target DNN Instance.
The Persona Bar -> Themes flickers for a moment, but then is a blank, empty div in the iframe for Themes.  This prevents the ability to fix the default theme for the newly created site. 
Updating the home page to an installed theme to resolve errors on page about missing theme did not have any effect on Persona  Bar -> Themes.

Similar to https://github.com/dnnsoftware/Dnn.Platform/issues/3224 but while 3224 was an edge case (direct edit/delete of file), this problem is replicatable through the DNN UI; when a Site Template is imported, but the theme used is not present on the DNN Instance.

workaround:
While a DBA can fix the entry for the Default Theme, this is not always an available option.  Having the Persona Bar -> Theme recover gracefully when the default theme for a site is missing will allow the problem to be fixed through the UI.

## Steps to reproduce
* Create new Empty Site
* Install theme
* Set the new site's default theme to the one installed
* Create Site Template   
* Export the Site Template
* update the empty site  to use other theme for default
* Uninstall Theme
* Import Site Template : import succeeds without error, home page has error trying to ref the missing theme, open personabar  -> themes, but cannot edit themes to resolve default template.  

alternate steps to reproduce
* have 2 dnn instances
* Create Site Template on first DNN instance with custom theme
* Create Site Template
* Export the template
* Import to second DNN instance without the custom theme : import succeeds without error, home page has error trying to ref the missing theme, open personabar  -> themes, but cannot edit themes to resolve default template.  

## Current behavior
Persona Bar -> Themes flashes then is empty as the iframe fails to display anything due to js error 

## Expected behavior
Themes pane displays with error about 'missing theme', so a replacement theme can be chosen (recover)

## Screenshots
<img width="675" alt="browserErrors" src="https://user-images.githubusercontent.com/40669744/95091302-46876f80-06f4-11eb-9bf3-bb381b363d9f.png">

## Error information
export-bundle.js throws an error in the browser console : Cannot read property 'themeName' of nulll at t.value

## Additional context
Provide any additional context that may be helpful in understanding and/or resolving the bug.

## Affected version
9.6.2

## Affected browser
not applicable

**Comment by Timo-Breumelhof** [comment]
Right, I logged https://github.com/dnnsoftware/Dnn.Platform/issues/3224 but it's an issue when importing a Site template

**Comment by RockSolidSW** [comment]
Is this issue in queue to be addressed?  We are experiencing it as well. Thanks!

**Comment by valadas** [comment]
I have triaged the issue, now since it's open source, we just need a contributor to tackle it and send a PR...
